scripts/gdb_printers.py

```diff
@@ -59,5 +61,259 @@ def build_pretty_printer():
     pp.add_printer('TurnLaneData', '::TurnLaneData$', TurnLaneDataPrinter)
     return pp
 
-#gdb.pretty_printers = [filter(lambda x: x.name != 'OSRM', gdb.pretty_printers)]
```

Comment:
it is needed to reload pretty printers, otherwise gdb module throws `RuntimeError: pretty-printer already registered: OSRM`. I will keep it commented, because it is needed only during script development